include/mata/nfa/delta.hh

```diff
@@ -444,7 +494,9 @@ public:
      * @return An iterator to @c SymbolPost with epsilon symbol. End iterator when there are no epsilon transitions.
      */
     static StatePost::const_iterator epsilon_symbol_posts(const StatePost& state_post, Symbol epsilon = EPSILON);
-}; // struct Delta.
+private:
+    std::vector<StatePost> state_posts_;
+}; // class Delta.
```

Comment:
Why was this moved to the end of class definition?

Reply:
I needed more private members earlier (which are not needed now), which had to be defined only after their types have been defined in the Delta class. Therefore, the private members list had to be at the end of the class definition. In the end, only the original member remained, but private members (be it member variables or functions), should not be the first thing you see when you are looking at a class interface, anyway. Henceforth, I decided to keep this change as it is. It helps with readability: the user of Delta does not need to bother themselves with the knowledge that internally, there is some vector of state posts in `Delta`, which they cannot access or do anything with, anyway.